roofit/roofitcore/test/TestStatistics/testLikelihoodJob.cxx

```diff
@@ -109,7 +109,8 @@ TEST_F(LikelihoodJobTest, UnbinnedGaussian1D)
 {
    std::tie(nll, pdf, data, values) = generate_1D_gaussian_pdf_nll(w, 10000);
    likelihood = RooFit::TestStatistics::buildLikelihood(pdf, data);
-   auto nll_ts = LikelihoodWrapper::create(RooFit::TestStatistics::LikelihoodMode::multiprocess, likelihood, clean_flags);
+   auto nll_ts =
+      LikelihoodWrapper::create(RooFit::TestStatistics::LikelihoodMode::multiprocess, likelihood, clean_flags);
```

Comment:
Perhaps it would be better to change this pure style change and similar ones below back to not bury the actual changes in the noise.

Reply:
Yes you're right, thanks for spotting that! I haven't realized the file was not completely formatted before